### PR TITLE
remove references to `System.IdentityModel.Tokens.Jwt` in JWT token generation

### PR DESCRIPTION
**Pull request by xingsy97**
### Summary of the changes
 - remove references to System.IdentityModel.Tokens.Jwt in JWT token generation

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -79,7 +79,7 @@
     <SignedPackageFile Include="$(PublishDir)Newtonsoft.Json.dll" Certificate="$(AssemblySigningCertName)" Visible="false" />
     <SignedPackageFile Include="$(PublishDir)System.IO.Pipelines.dll" Certificate="$(AssemblySigningCertName)" Visible="false" />
     <SignedPackageFile Include="$(PublishDir)System.IdentityModel.Tokens.Jwt.dll" Certificate="$(AssemblySigningCertName)" Visible="false" />
-    <SignedPackageFile Include="$(PublishDir)System.Text.Encodings.Web.dll" Certificate="$(AssemblySigningCertName)" Visible="false" />
+	  <SignedPackageFile Include="$(PublishDir)System.Text.Encodings.Web.dll" Certificate="$(AssemblySigningCertName)" Visible="false" />
```
Comment (vicancy) [inline Directory.Build.props]: revert?

File: src/Microsoft.Azure.SignalR.Common/Auth/JwtSecurityTokenHandlerSignalR.cs [hunk 2]
```diff
@@ -20,9 +18,9 @@
 
 namespace Microsoft.Azure.SignalR
 {
-    internal class JwtSecurityTokenHandlerSignalR : JwtSecurityTokenHandler
+    internal class JwtSecurityTokenHandlerSignalR
```
Comment (vicancy) [inline src/Microsoft.Azure.SignalR.Common/Auth/JwtSecurityTokenHandlerSignalR.cs:21]: naming `SignalRJwtSecurityTokenHandler`

File: src/Microsoft.Azure.SignalR.Common/Auth/JwtSecurityTokenHandlerSignalR.cs [hunk 3]
```diff
@@ -57,8 +55,9 @@ internal class JwtSecurityTokenHandlerSignalR : JwtSecurityTokenHandler
             JwtPayload payload = new JwtPayload(issuer, audience, (subject == null ? null : OutboundClaimTypeTransform(subject.Claims)), notBefore, expires, issuedAt);
             JwtHeader header = new JwtHeader(kid, algorithm);
 
-            if (subject?.Actor != null)
-                payload.AddClaim(new Claim(JwtRegisteredClaimNames.Actort, CreateActorValue(subject.Actor)));
+            // Because subject.Actor is always null, we skip this
```
Comment (vicancy) [inline src/Microsoft.Azure.SignalR.Common/Auth/JwtSecurityTokenHandlerSignalR.cs:58]: remove code, and merge the comment with the class comment above

Comment (xingsy97) [reply src/Microsoft.Azure.SignalR.Common/Auth/JwtSecurityTokenHandlerSignalR.cs]: fixed

File: src/Microsoft.Azure.SignalR.Common/Auth/JwtSecurityTokenHandlerSignalR.cs (hunk 2 above)
Comment (xingsy97) [reply src/Microsoft.Azure.SignalR.Common/Auth/JwtSecurityTokenHandlerSignalR.cs]: Renamed

File: Directory.Build.props (hunk 1 above)
Comment (xingsy97) [reply Directory.Build.props]: This file is no long in [Files changed list](https://github.com/Azure/azure-signalr/pull/1657/files)

**Review by vicancy — APPROVED** [review]

**Pull request merged**